### ISSUE TITLE
RoundedTB sometimes fails to load settings

### ISSUE THREAD

**Issue by Ovyerus** [torchgm/RoundedTB#30]
(Installed through the Microsoft Store)

Sometimes on computer boot, RoundedTB fails to load settings from `rtb.json` and sets margin and corner radius both to `0`. `rtb.json` does exist (although didn't check when the issue actually occurred due to not knowing about its existence, will check again if it re-occurs) and is properly accessible since the app was installed through the MS Store.

Image of file properties for `rtb.json`:
![properties window for rtb.json](https://fire.is-fi.re/9iHHwaK.png)

**Comment by torchgm** [comment]
Hi Ovy, apologies for taking so long to get around to this - does this still occur in R3 on the Store?

**Comment by Ovyerus** [comment]
I don't believe I've had this re-occur since the last couple incidents around the time of opening the issue, haven't since latest update though.

**Comment by torchgm** [comment]
Hm, alrighty - I'll close this for now because this is the only report I've had of it and I've not encountered it myself either (even after trying to induce it in various manners), so I'm hedging my bets on an external issue for now. Please do reopen it if it occurs again though!
